docs/integrations/spark/main_concept.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 1
+title: Main Concepts
```

Comment:
Not sure about the title, but have no alternative idea.

Reply:
Merging for now. How about `Architecture` ?